<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonComponentClass</class>
 <widget class="QDialog" name="CommonComponentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommonComponent</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>SpinBox数值显示</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>数量：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="suffix">
           <string>kg</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>212</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>单价：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="prefix">
           <string>$</string>
          </property>
          <property name="singleStep">
           <double>2.000000000000000</double>
          </property>
          <property name="value">
           <double>32.880000000000003</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>QSlider滑动块</string>
      </attribute>
      <widget class="QDial" name="dial">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>34</number>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>160</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>34</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>160</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>162</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>160</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>49</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>40</y>
         <width>19</width>
         <height>160</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>167</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>320</y>
         <width>401</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>20</y>
         <width>16</width>
         <height>261</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>160</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>176</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>QProgressBar进度条</string>
      </attribute>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>341</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>341</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>67</number>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>341</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>32</number>
       </property>
       <property name="format">
        <string>%m</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>311</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>87</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>%m</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>QLCDNumber数字显示</string>
      </attribute>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>311</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Aharoni</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="smallDecimalPoint">
        <bool>true</bool>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
       <property name="value" stdset="0">
        <double>3539.590239999999994</double>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>QDateTimeEdit日期时间</string>
      </attribute>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="firstDayOfWeek">
        <enum>Qt::Monday</enum>
       </property>
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QCalendarWidget::SingleSelection</enum>
       </property>
       <property name="horizontalHeaderFormat">
        <enum>QCalendarWidget::ShortDayNames</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::ISOWeekNumbers</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>QTimer定时器</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLCDNumber" name="lcdHour">
          <property name="font">
           <font>
            <family>Aharoni</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Aharoni</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdMin">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>43</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Aharoni</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdSec">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Filled</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>28</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QProgressBar" name="pbTimer">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>331</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStart">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStop">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>200</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>QComboBox下拉框</string>
      </attribute>
      <widget class="QComboBox" name="cbClass">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>190</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>语文</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>数学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>英语</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>物理</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>化学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>地理</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cbCity">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>北京</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上海</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>广州</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>深圳</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>QPlainTextEdit多行文本</string>
      </attribute>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>321</width>
         <height>221</height>
        </rect>
       </property>
       <property name="plainText">
        <string>12132343
2342356575
2453645646457675
节地埒肝一 五
</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CommonComponent.qrc"/>
 </resources>
 <connections/>
</ui>
